pkg/build/executor.go

```diff
@@ -21,13 +21,24 @@ import (
 	"github.com/pkg/errors"
 )
 
-func Execute(name string, arg ...string) error {
+func (bc *Context) Execute(name string, arg ...string) error {
+	logname := name
+
+	if bc.UseProot {
+		arg = append(arg, "")
+		arg = append(arg, "")
```

Comment:
If I'm reading this right, you're shifting the arguments back two so you can prefix them with `-0 name`. It might be a bit simpler to read if it's setup like:

`arg = append([]string{"-0", "name"}, arg...)`
https://go.dev/play/p/eCmQi956x1T

Reply:
Thanks!